content/scipy/concepts/scipy-stats/terms/descriptive-stats/descriptive-stats.md

```diff
@@ -0,0 +1,65 @@
+---
+Title: 'Descriptive Stats'
+Description: 'Refers to the process of summarizing and describing the essential features of a dataset.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Data'
+  - 'Functions'
+  - 'Math'
+  - 'Python'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+In SciPy, **descriptive stats** refers to the process of summarizing and describing the essential features of a dataset. It enables users to understand the basic characteristics of the given data through various metrics such as central tendency, variability, and distribution shape.
```

Comment:
This can be rewritten for better readability like this -

```suggestion
In SciPy, **descriptive statistics** refers to summarizing and analyzing a dataset's key characteristics. It helps summarize essential properties such as central tendency, variability, and distribution shape.
```